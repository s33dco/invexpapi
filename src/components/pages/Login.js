import React, { Fragment } from 'react';

const Login = () => {
	return (
		<Fragment>
			<p>Login</p>
		</Fragment>
	);
};

export default Login;
