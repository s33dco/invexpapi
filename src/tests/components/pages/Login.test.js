import React from 'react';
import { shallow } from 'enzyme';
import Login from '../../../components/pages/Login';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<Login />);
});

describe('Login component', () => {
	it('should render Login component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
