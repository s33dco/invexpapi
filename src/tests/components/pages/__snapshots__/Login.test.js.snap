// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component should render Login component correctly 1`] = `ShallowWrapper {}`;
